gulp/tasks/test.js

```diff
@@ -406,18 +406,18 @@ function spawnProtractor( ) {
 }
 
 /**
- * Run coveralls reports on Travis.
+ * Run codecov reports on Travis.
  */
-function testCoveralls() {
+function testCodecov() {
   gulp.src( configTest.tests + '/unit_test_coverage/lcov.info' )
-    .pipe( gulpCoveralls() );
+    .pipe( gulpCodecov() );
 }
 
 gulp.task( 'test', [ 'lint', 'test:unit' ] );
 gulp.task( 'test:a11y', testA11y );
 gulp.task( 'test:acceptance', testAcceptanceBrowser );
 gulp.task( 'test:acceptance:protractor', spawnProtractor );
-gulp.task( 'test:coveralls', testCoveralls );
+gulp.task( 'test:codecov', testCodecov );
```

Comment:
Is there a tool-agnostic name we could give this task? `gulp test:cloud` perhaps? Any other ideas?

Reply:
`test:submit-coverage`?

Reply:
It's kiiiiinda long

Reply:
But nice and descriptive!